cosmopolitan_continents/migrations/0001_initial.py

```diff
@@ -11,9 +11,9 @@ class Migration(migrations.Migration):
 
     operations = [
         migrations.CreateModel(
-            name='Continent',
+            name='CosmopolitanContinent',
             fields=[
-                ('id', models.CharField(serialize=False, primary_key=True, max_length=2)),
+                ('id', models.CharField(max_length=2, serialize=False, primary_key=True)),
```

Comment:
Also add the 3 letter variant too. Maybe an array field called `codes` which lists all the codes for the country (including the one we use as id).